src/ProjectTemplates/BlazorTemplates.Tests/BlazorServerTemplateTest.cs

```diff
@@ -27,7 +27,6 @@ public BlazorServerTemplateTest(ProjectFactoryFixture projectFactory, BrowserFix
         public Project Project { get; private set; }
 
         [Fact]
-        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/20172")]
```

Comment:
nit: you aren't supposed to remove this after potentially fixing a test, you should wait at least 30 days and whoever is on build ops will eventually remove it from quarantine if it's been passing